test/features/10-functional/10-basic/40-service-details.feature

```diff
@@ -12,3 +12,20 @@ Feature: Service details page
     Scenario: The description is limited to a set of dot points
         When I visit /service/13841
         Then I should see "Advice on legal matters"
+
+    Scenario: The address is a link to google maps
+        When I visit /service/866464
+        Then I can get to google maps by clicking "33 Elizabeth Street Richmond Vic 3121"
+
+    Scenario: The phone numbers (without fax) are available
+        When I visit /service/866464
+        Then I should see the phone numbers
+        -------------------------------------------
+        Phone (03) 9418 7447
+        -------------------------------------------
+        When I click on "Other contact options"
+        Then I should see the phone numbers
+        -------------------------------------------
+        Phone (03) 9418 7447
+        Phone 1300 306 645
```

Comment:
Can we make this a test for phone type priority (i.e. freecall > phone > mobile)